dependency-management/dependency-management-impl/src/main/java/io/cloudslang/dependency/impl/services/DependencyServiceImpl.java

```diff
@@ -99,7 +128,10 @@ private void initMaven() throws ClassNotFoundException, NoSuchMethodException, M
 
     private void initMavenLogs() {
         File mavenLogFolderFile = new File(new File(calculateLogFolderPath()), MavenConfig.MAVEN_FOLDER);
-        mavenLogFolderFile.mkdirs();
+        boolean dirsCreated = mavenLogFolderFile.mkdirs();
+        if (!dirsCreated) {
+            logger.error("Failed to create maven log directories " + mavenLogFolderFile.getAbsolutePath());
+        }

```

Comment:
Not part of your change, but line 167 is very inneficient, we should change it to use equalsIgnoreCase